api/services/PassportService.js

```diff
@@ -203,11 +203,12 @@ module.exports = class PassportService extends Service {
 
     criteria[fieldName] = identifier
 
-    return this.app.services.FootprintService.find('User', criteria, {populate: 'passports', findOne: true})
+    return this.app.services.FootprintService.find('User', criteria, {populate: 'passports'})
```

Comment:
Used some months ago for an internal angular2 cms, initially it's strange but when understood becomes very powerful